tsconfig.json

```diff
@@ -5,6 +5,5 @@
     "noImplicitOverride": true,
     "strictNullChecks": true,
     "resolveJsonModule": true
-  },
-  "exclude": ["tests/integration", "tests/esm", "tests/vitest"]
```

Comment:
probably the reason for that problem is that I made top level tsconfig too inclusive

Reply:
nope, tried this, didn't help